clash-ghc/src-ghc/Clash/GHC/Util.hs

```diff
@@ -9,8 +12,12 @@ import GHC.Exception      (SomeException)
 import System.Exit        (ExitCode(ExitFailure), exitWith)
 
 import Clash.Util         (ClashException(..))
+import Clash.Util.Interpolate (i)
 import Clash.Driver.Types (ClashOpts(..))
 
+text' :: String -> SDoc
```

Comment:
```suggestion
-- | Like 'lines', but returning a horizontally spaced SDoc instead of a list:
--
-- >>> textLines "a\nb"
-- a $$ b
textLines :: String -> SDoc
```

Reply:
Done